kotest-property/src/commonMain/kotlin/io/kotest/property/arbitrary/arbs.kt

```diff
@@ -40,7 +49,7 @@ fun <A> arb(edgecases: List<A> = emptyList(), f: (RandomSource) -> Sequence<A>)
  * supplied edge case values, and provides values from sequence returning function.
  */
 @Deprecated(
-   "This function will no longer accept Sequence<A>. Use Arb.create or arb with (RandomSource -> A) for compatibility"
+   "This function will no longer accept Sequence<A>. Use Arb.create or arb with (RandomSource -> A) for compatibility. This function will be removed in 4.5."
```

Comment:
Should we consider adding `replaceWith` argument to `@Deprecated` that will help IDE to do code assistance in replacing it with new api.

[Reference](https://blog.8bitzen.com/posts/18-05-2018-kotlin-deprecated-replace-with/#:~:text=Anyway%20in%20Kotlin%20there%20is,to%20replace%20it%20with%3A%20replaceWith%20.)

Reply:
We can't because the required code is different. Users need to modify their logic.